js/Common.js

```diff
@@ -256,7 +256,7 @@ if (!CRM.vars) CRM.vars = {};
         CRM.CMSjQuery = window.jQuery;
         window.jQuery = CRM.$;
       }
-      script.src = url;
+      script.src = url + (_.includes(url, '?') ? '&' : '?') + CRM.config.resourceCacheCode;
```

Comment:
To be consistent with the other cache-busting code in Civi, this should probably use `?r=xyz` (with `r=`).

Otherwise, looks good.

Reply:
Done

Reply:
Patch pushed within 45s of the comment. That might be a record.